tests/tacacs/utils.py

```diff
@@ -289,3 +290,30 @@ def check_server_received(ptfhost, data):
     logger.info(sed_command)
     logger.info(res["stdout_lines"])
     pytest_assert(len(res["stdout_lines"]) > 0)
+
+
+def get_latest_aaa_config_update_log(duthost):
+    res = duthost.command("sed -nE '/hostcfgd: AAA Update/P' /var/log/syslog")
```

Comment:
>syslog [](http://example.com/codeflow?start=70&length=6)

Repeatedly sed the syslog file is not robust. For example, you may miss some logging if syslog is just rotated. `aaa_update` will modify the some config files. Is it better to monitor those files?

Reply:
Fixed, check config is not accurate because auditd need take ~1 second to reload config file, change to check auditd service status.